code/modules/research/designs/mechfab_designs.dm

```diff
@@ -891,7 +891,7 @@
 /datum/design/rig
 	build_type = MECHFAB
 	category = "RIG Assemblies"
-	materials = list(DEFAULT_WALL_MATERIAL = 25000, "glass" = 10000)
+	materials = list(DEFAULT_WALL_MATERIAL = 25000, "glass" = 10000, "silver" = 5500, "gold" = 5500)
```

Comment:
this makes EVA rig require silver and RIG. I intentionally wanted EVA rigs to be printed without requiring mining materials.

Reply:
I am not for rigs being that easily available. They were always meant to come in a limited number or in rare cases. I think that having the basic ones require a bit of material should keep this theme a bit.